src/olympia/devhub/helpers.py

```diff
@@ -115,7 +115,17 @@ def docs_breadcrumbs(context, items=None):
 @jinja2.contextfunction
 def add_file_modal(context, title, action, action_label, modal_type='file'):
     addon = context['addon']
-    upload_url = reverse('devhub.upload_for_addon', args=[addon.slug])
+    version = context.get('version',
+                          addon.find_latest_version_including_rejected())
```

Comment:
It's currently used both for new version uploads (where there isn't a version, natch); and for new file uploads where there is.  The fallback is for new version and will be there until the new version upload flow is live;  the rest of the function will be there until new file upload is live.